custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2021,11 +1948,53 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, $fil
                 },
             },
             yAxis1: {
+                axisLabel: 'Height (Cms)',
```

Comment:
Was this in the spec like this? I think generally it would be "(cm)" but I'm not familiar with Indian measurement conventions.

Reply:
@emord I don't know why, I'm also not familiar with Indian measurement conventions., I added as Lilianna requested in this case: https://manage.dimagi.com/default.asp?262726#1400588